.github/workflows/weekly.yaml

```diff
@@ -17,13 +17,8 @@ jobs:
       - name: Install Tools & Dependencies
         uses: ./.github/actions/install
 
-      - name: Run update:integrations script
-        run: pnpm run update:integrations
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Run update:themes script
```

Comment:
all theme updates are safe since we manually add/remove them, moved this into the nightly job